skops/hub_utils/_hf_hub.py

```diff
@@ -755,9 +755,9 @@ def get_model_output(repo_id: str, data: Any, token: Optional[str] = None) -> An
         inputs = {f"x{i}": data[:, i] for i in range(data.shape[1])}
         inputs = {"data": inputs}
 
-    res = InferenceApi(repo_id=repo_id, task=model_info.pipeline_tag, token=token)(
-        inputs=inputs
-    )
+    client = InferenceClient(token=token)
+    res_bytes = client.post(json={"inputs": inputs}, model=repo_id)
+    res = json.loads(res_bytes.decode("utf-8"))
```

Comment:
don't we have a test for this?

Reply:
Yes, I forgot that we require `[CI inference]` in the commit message to trigger it. This is done now but of course we run into the rate limit again...

Reply:
Should we remove the functionality then since the rate limit is so low?

Reply:
It's mostly an issue for our CI, right? Which is probably why we made the test opt-in in the first place. I'd say we leave it, in case some users actually use the feature. If we want to remove it later on, we can do it properly with a deprecation cycle.

Reply:
We can also add a random sleep between 1 and 100 hours to avoid hitting the limit ;-P

Reply:
Hmm, not happy with a functionality that even our CI hits the limits pretty much every time. But we can merge and figure that out in a separate PR.